microgen/boxMesh.py

```diff
@@ -0,0 +1,586 @@
+"""
+Cubic mesh for FE
+"""
+
+from typing import Sequence, Optional, Union
+
+import pyvista as pv
+import numpy as np
+import warnings
+from scipy.spatial import KDTree
+from .rve import Rve
+from .singleMesh import SingleMesh, _check_if_only_linear_tetrahedral
+
+# try: 
```

Comment:
This is in case we would like to utilise multi_ray_tracing again. However it seems that the try was not properly working on MacOS, so I left it there as a reminder